pravega/src/test/scala/docs/scaladsl/PravegaSinkSpec.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.scaladsl
+
+import akka.stream.alpakka.pravega.ReaderSettings
+
+import akka.stream.alpakka.pravega.scaladsl.Pravega
+import akka.stream.scaladsl.Source
+
+import io.pravega.client.stream.EventWriterConfig
+import io.pravega.client.stream.impl.JavaSerializer
+
+class PravegaSinkSpec extends PravegaBaseSpec {
+
+  final implicit val eventWriterConfig = EventWriterConfig.builder().build()
+
+  implicit val serializer = new JavaSerializer[String]
+  implicit val settings = new ReaderSettings(groupName, readerId, serializer)
+
+  "Pravega connector" should {
+    "compiles settings" in {
+      new ReaderSettings(groupName, readerId, serializer)
+    }
+
+    "compiles sources" in {
+
+      Pravega.source(scope, streamName)
+    }
+
+    "runs sources" in {
+
+      // #writing
+
+      val fut = Source(1 to 10).map(i => s"event_$i").runWith(Pravega.sink(scope, streamName))
```

Comment:
It would be nice to have a "there and back again" style of integration test which tests the sink and source together.